src/App.css

```diff
@@ -119,29 +133,14 @@
     padding: 0px;
 }
 
-.ap-list {
-    margin-top: -7vh;
-    background-color: white;
-    border-radius: 5px;
-    height: 70vh;
-    margin-left: auto;
-    margin-right: auto;
-    padding: 0px;
-}
-
 .btn-primary, .btn-primary:hover, .btn-primary:active, .btn-primary:visited {
     background-color: #db2cc4 !important;
     border-color: #db2cc4 !important;
```

Comment:
Could you add `float: right` back. Otherwise, the button looks strange on a wide monitor, and it will cause it to clip off the edge if the size is too small.

Reply:
I removed the left internal padding on first column, and then the button div using the property text-align got move to right. This way we have same padding both sides(20px).